SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -817,6 +875,67 @@ void ASpatialFunctionalTest::OnReplicated_bFinishedTest()
 	}
 }
 
+void ASpatialFunctionalTest::LogAndClearRequireHandler()
+{
+	TArray<FSpatialFunctionalTestRequire> RequiresOrdered = RequireHandler.GetAndClearStepRequires();
+
+	ASpatialFunctionalTestFlowController* FlowController = GetLocalFlowController();
+	if (FlowController)
+	{
+		const FString& WorkerName = FlowController->GetDisplayName();
+
+		for (const auto& Require : RequiresOrdered)
+		{
+			FString Msg;
+			if (Require.bPassed)
+			{
+				Msg = FString::Printf(TEXT("%s [Passed] %s : %s"), *WorkerName, *Require.Msg, *Require.StatusMsg);
+			}
+			else
+			{
+				Msg = FString::Printf(TEXT("%s [Failed] %s : %s"), *WorkerName, *Require.Msg, *Require.StatusMsg);
+			}
+
+			FlowController->ServerNotifyLogRequireMessages(*Msg, Require.bPassed);
```

Comment:
```suggestion
			const FString Msg = = FString::Printf(TEXT("%s [%s] %s : %s"), *WorkerName, ((Require.bPassed? : "Passed" : "Failed")), *Require.Msg, *Require.StatusMsg);
			FlowController->ServerNotifyLogRequireMessages(*Msg, Require.bPassed);
```

Reply:
Discussion moved to the RequireHandler file